Source/WebCore/page/LocalDOMWindow.cpp

```diff
@@ -1612,7 +1612,8 @@ Ref<CSSStyleDeclaration> LocalDOMWindow::getComputedStyle(Element& element, cons
     auto pseudoId = CSSSelector::parsePseudoElement(pseudoElt, CSSSelectorParserContext { element.document() });
     if (!pseudoId)
         return CSSComputedStyleDeclaration::createEmpty(element);
-    return CSSComputedStyleDeclaration::create(element, Style::PseudoElementIdentifier { *pseudoId });
```

Comment:
We need to fix the code to avoid having so many different invalid states there (`std::nullopt` + PseudoId::None`)

Reply:
Agreed, changing `CSSSelector::parsePseudoElement` will happen at some point.